RIPS/rip-7560.md

```diff
@@ -217,10 +225,20 @@ struct TransactionType4 {
     bytes paymasterData;
     bytes callData;
     bytes signature;
+    address[] authorizationList;
+    bool[] authorizationListStatus;
```

Comment:
maybe better to provide `authorizedList` and `failedAuthorizationList` (which is expected to be empty)

Reply:
Just talked to @shahafn about that - we could not see how would it make it better? Appears to provide the exact same information (except, maybe, losing the ordering - which might even be a downside in some edge-cases) and not improving anything.
I prefer the `bool[]` approach.